test/File.jl

```diff
@@ -36,6 +36,14 @@
     )
 
     @test get_metadata_path(my_file) == metadata_path
+    @test get_sub(my_file) == "test"
+    @test get_ses(my_file) == "1"
+    @test get_sub(
+        dirname(my_file.path)*Base.Filesystem.path_separator, from_fname=false
+    ) == "subtest"
+    @test get_ses(
+        dirname(my_file.path)*Base.Filesystem.path_separator, from_fname=false
+    ) == "1"
 end
```

Comment:
Could do with a test for `extract_from_full_path`?

Reply:
Hmm actually the `extract_from_full_path` param defaults to `true` so this test case already covers that:

https://github.com/TRIImaging/BIDSTools.jl/blob/fa5586dc71a320de840a646e1cfe7e359612e218/test/File.jl#L26

Reply:
But do we actually hit the code path for extracting from the `path`? It looks like we don't because the example file name already has `sub` and `ses` included?

Reply:
You're right, I'll add that test!